packages/video_player/video_player/android/build.gradle

```diff
@@ -1,6 +1,5 @@
 group 'io.flutter.plugins.videoplayer'
 version '1.0-SNAPSHOT'
-def args = ["-Xlint:deprecation","-Xlint:unchecked","-Werror"]
```

Comment:
why remove this?

Reply:
Embarrassed several developers and several issues for-werror. I want to keep it but I don't want to block developers.

Reply:
We should probably split this into separate patches to avoid a revert catching both when it only intends to get one. Otherwise LGTM